tasks/post_to_queue_and_trigger_step_function/sqs_library.py

```diff
@@ -0,0 +1,140 @@
+"""
+Name: sqs_library.py
+Description: library for post_to_queue_and_trigger_step_function lambda function for posting to fifo SQS queue.
```

Comment:
Per our discussion on FIFO and how a FIFO DLQ we think will break the issue where processing would stop because order must be maintained, will leave as FIFO for now until we see some real world use/testing.